core/dovecot/start.py

```diff
@@ -32,6 +33,15 @@ def start_podop():
 for dovecot_file in glob.glob("/conf/*.conf"):
     conf.jinja(dovecot_file, os.environ, os.path.join("/etc/dovecot", os.path.basename(dovecot_file)))
 
+try:
+    os.mkdir("/conf/bin")
+except FileExistsError:
+    pass
+for script_file in glob.glob("/conf/*.script"):
```

Comment:
We should not write these files directly to the source directory: everything in that directory is seen as executable in sieve scripts. I guess there is very little chance the source jinja contains an exploitable vulnerability without actually passing to a jinja parser, but you never really are too safe with whitelists.

Reply:
Never mind, it seems I can't read, or distinguish /conf from /conf/bin :unamused: